pkg/minikube/cruntime/cruntime.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cruntime contains code specific to container runtimes
+package cruntime
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+const MinikubeContainerPrefix = "k8s_"
+
+// CommandRunner is the subset of bootstrapper.CommandRunner this package consumes
+type CommandRunner interface {
+	Run(string) error
+	CombinedOutput(string) (string, error)
+}
+
+// Manager is a common interface for container runtimes
+type Manager interface {
```

Comment:
I would love to name this `Runtime` - I find myself constantly translating to "Runtime" as I read "Manager".
It would create a bit of stuttering...but it just makes more sense to me. WDYT?